internal/api/graphql/graph/resolver/component_instance_filter.go

```diff
@@ -44,6 +44,51 @@ func (r *componentInstanceFilterValueResolver) Ccrn(ctx context.Context, obj *mo
 	return item, err
 }
 
+func (r *componentInstanceFilterValueResolver) Region(ctx context.Context, obj *model.ComponentInstanceFilterValue, filter *model.ComponentInstanceFilter) (*model.FilterItem, error) {
+	item, err := baseResolver.RegionBaseResolver(r.App, ctx, filter)
+	if err != nil {
+		return nil, err
+	}
+	item.FilterName = &baseResolver.FilterDisplayRegion
```

Comment:
Here you don't use the display name but, the `ComponentInstanceFilterRegion` name.

The API will return:

```
{
  "data": {
    "ComponentInstanceFilterValues": {
      "region": {
        "filterName": "region",
        "displayName": "Region"
      },
      "domain": {
        "filterName": "domain",
        "displayName": "Domain"
      },
      "cluster": {
        "filterName": "cluster",
        "displayName": "Cluster"
      }
    }
  }
}
```